js/textbox/shared/Textbox.svelte

```diff
@@ -68,6 +68,7 @@
 			scroll();
 		}
 		value_is_output = false;
+		el.focus();
```

Comment:
Should this be at line 114 instead? Right now, it causes any change in the textbox value to focus on the textbox. Which means, e.g. if you launch 

```py
import gradio as gr

gr.Interface(lambda x:x, "textbox", "textbox").launch()
```

it focuses on the input textbox, which is not desired